src/component/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,58 @@
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  onDelete: (id: number) => void;
+  idTodo: number[];
+};
+
+export const ToDoList: React.FC<Props> = ({ todos, onDelete, idTodo }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(({ title, id, completed }) => (
+        <div
+          data-cy="Todo"
+          className={classNames('todo', { completed: completed })}
+          key={id}
+        >
+          {/* eslint-disable jsx-a11y/label-has-associated-control  */}
+          <label className="todo__status-label" htmlFor={'' + id}>
+            <input
+              id={'' + id}
```

Comment:
why you need this ''? 
If you really need sting you can use String(id). 
But I don`t see any reason for it. Maybe I miss something...